presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -370,7 +371,7 @@ public Void visitGroupId(GroupIdNode node, Void context)
         @Override
         public Void visitFilter(FilterNode node, Void context)
         {
-            String expression = formatter.formatRowExpression(node.getPredicate());
+            String expression = isExpression(node.getPredicate()) ? node.getPredicate().toString() : formatter.formatRowExpression(node.getPredicate());
```

Comment:
`GraphvizPrinter` (and all other plan printer) should not expect an `Expression`. Rather than having `isExpression` here, let's fix the tests if they are failing.